src/parser/mod.rs

```diff
@@ -1323,13 +1324,26 @@ impl<'a> Parser<'a> {
                 expr: Box::new(expr),
                 trim_where,
                 trim_what: Some(trim_what),
+                trim_characters: None,
+            })
+        } else if self.consume_token(&Token::Comma)
+            && dialect_of!(self is SnowflakeDialect | BigQueryDialect)
```

Comment:
I think the general pattern for this crate is to also include `GenericDialect` so it is a superset (as much as possible) of all dialects

Reply:
@alamb .. added `GenericDialect`